chatstripe/payment/models.py

```diff
@@ -4,15 +4,15 @@
 # Create your models here.
 
 class CardDetail(models.Model):
-    user = models.ForeignKey(User, on_delete=models.SET_NULL)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
     card_number = models.CharField(max_length = 16)
-    card_type = (
+    card_type_choices = (
         ('mastercard', _('mastercard')),
         ('visa', _('visa')),
         ('american_express', _('American Express')),
         ('discover', _('Discover'))
     )
-    card_type = models.CharField(max_length = 30)
+    card_type = models.CharField(max_length = 30, choices=card_type_choices, default="mastercard")
```

Comment:
don't provide a default option in card_type. if nothing is provided the card will be considered as MasterCard, which is a wrong assumption. The card can be of any type.